bin/import_swb.py

```diff
@@ -13,32 +17,88 @@
 import unicodedata
 import wave
 import codecs
+import tarfile
+#from util.downloader import maybe_download, SIMPLE_BAR
```

Comment:
Left overs? This should be removed.

Reply:
Thanks, I have removed this, added pep8 spacing between functions and added a better run comment.